.travis.yml

```diff
@@ -12,6 +12,11 @@ matrix:
         - DOCKERIMAGE=linux-anvil-comp7
         - DOCKERTAG=jnlp-slave
 
+    - os: linux
+      env:
+        - DOCKERIMAGE=linux-anvil-cos7-comp7
```

Comment:
```suggestion
        - DOCKERIMAGE=linux-anvil-cos7
```

Reply:
@isuruf - can you clarify please? Are you suggesting to change the name to `linux-anvil-cos7` everywhere?

Reply:
Yes